src/UserInterface.js

```diff
@@ -1,41 +1,72 @@
 /*
  * Default UI for each image-sequencer module
  */
-module.exports = function UserInterface(options) {
-
-  options = options || {};
-  options.container = options.container || ".panels";
-  options.id = options.id;
-  options.instanceName = options.instanceName;
-  options.random = options.random || parseInt(Math.random() * (new Date()).getTime() / 1000000);
-  options.uniqueSelector = options.uniqueSelector || options.selector + '-' + options.random;
-  $(options.container).append('<div class="panel ' + options.selector + ' ' + options.uniqueSelector + '" id="sequencer-'+options.id+'"><div class="image"></div></div>');
-  options.el = options.el || $('.' + options.uniqueSelector);
-  createLabel(options.el);
-
-  // method to remove the UI for a given method, and remove the step
-  function display(image) {
-    options.el.find('.image').html(image);
-  }
+module.exports = function UserInterface(UI,options) {
 
-  // method to remove the UI for a given method, and remove the step
-  function remove() {
-    $('div#sequencer-'+options.id).remove();
-  }
+  return function userInterface(identity) {
 
-  // method to reorder steps, and update the UI
-  //function move() {}
+    var UI = UI || {};
 
-  function createLabel(el) {
-    if (options.title) el.prepend('<h3 class="title">' + options.title + '</h3> <button class="btn btn-default" onclick="'+options.instanceName+'.removeStep('+options.id+')">Remove Step</button>');
-  }
+    UI.onSetup = UI.onSetup || function() {
+      if(options.ui == "none") {
+        // No UI
+      }
```

Comment:
Yes, it would.
Actually for the unit tests, we started doing this:
```js
var sequencer = ImageSequencer({ui:"none"});
```

This is where it originated, I also feel we should make it `false`